universal-application-tool-0.0.1/conf/evolutions/default/26.sql

```diff
@@ -0,0 +1,8 @@
+# --- Add a column to applications that records the email address of the submitter
+# --- Also ensure that no global admins can be program admins.
```

Comment:
(blocking, maybe?) Does this "ensure that no global admins can be program admins" happen here?

Reply:
Done.